### DISCUSSION TITLE
Button (Text and Icon not aligned)

### DISCUSSION THREAD

**Discussion by mengtongun** [refinedev/refine#2321] [Q&A] [answered]
<img width="363" alt="image" src="https://user-images.githubusercontent.com/60088954/184535665-9ec7bf45-3ed8-4e1a-a5e6-48ab33992428.png">

We can see in the screenshot the icon is below the text.
The alignment is not equal. 
Please help 🙏🏻

**Comment by omeraplak** [comment, marked as answer]
Hey @mengtongun ,
Thank you for contacting us! Have you made any style changes (css import, custom theme, etc.) in app?

**Reply by mengtongun** [reply]
I have added TailwindCSS to app.

**Reply by mengtongun** [reply]
When I comment import css from tailwind the style is back. 
Do you have any solutions to use both Antd and TailwindCSS?

**Reply by omeraplak** [reply]
It might be because you're using Ant design and Tailwind together. Can you take a look here?  https://stackoverflow.com/questions/68924385/how-to-use-antdesign-with-tailwindcss-together-in-a-react-project/70451103

**Reply by mengtongun** [reply]
Thank you @omeraplak 
https://stackoverflow.com/questions/60853067/how-to-solve-ant-design-base-css-and-tailwindcss-base-css-alignments-problem/70532209#70532209
This helped me!
